sdk/textanalytics/Azure.AI.TextAnalytics/api/Azure.AI.TextAnalytics.netstandard2.0.cs

```diff
@@ -1171,7 +1109,7 @@ public static partial class TextAnalyticsModelFactory
         public static Azure.AI.TextAnalytics.TextAnalyticsOperationStatus Cancelling { get { throw null; } }
         public static Azure.AI.TextAnalytics.TextAnalyticsOperationStatus Failed { get { throw null; } }
         public static Azure.AI.TextAnalytics.TextAnalyticsOperationStatus NotStarted { get { throw null; } }
-        public static Azure.AI.TextAnalytics.TextAnalyticsOperationStatus PartiallySucceeded { get { throw null; } }
+        public static Azure.AI.TextAnalytics.TextAnalyticsOperationStatus PartiallyCompleted { get { throw null; } }
```

Comment:
PartiallyCompleted isn't a standard LRO status. At least in Python, we had to customize our pollers a bit to make sure that if this status was returned that we recognized it as a terminal state and stopped polling. Just calling out in case .NET has to do the same.

Reply:
There's a separate task for this I'll get in another PR. I'll be sure to clean this up with that.

Reply:
I left a comment on that separate issue: https://github.com/Azure/azure-sdk-for-net/issues/28356